### DISCUSSION TITLE
squatters

### DISCUSSION THREAD

**Discussion by osc2nuke** [NativePHP/laravel#259] [Ideas]
I'm up to creating a global website (app) and use NativePHP as a core to monitor houses via a webcam security system to prevent squatters take overs.
The idea is to use minimal resources "read: investments" and the police can be instantly contacted at the moment of 'burglary'.
For that i require sponsors in the security system field. ( Hardware testing ).
The MAIN idea of the concept is to create a niche in camera surveillance but also physically/virtual property protection, where the customer hires a contractor working from HOME. ( globally in any area of the planet)

Let me know if have shared ideas!
One of the reasons WHY i come up with this IDEA: https://www.youtube.com/watch?v=HFw7g_4LKPk

**Comment by simonhamp** [comment]
This Discussion category is more for ideas aimed specifically at NativePHP - not general ideas for products.
